backend/de.metas.global_qrcode.services/src/main/java/de/metas/global_qrcodes/service/GlobalQRCodeService.java

```diff
@@ -25,7 +26,11 @@
 @Service
 public class GlobalQRCodeService
 {
-	private static final AdProcessId default_qrCodeProcessId = AdProcessId.ofRepoId(584977); // hard coded process id
+	public static final String SYSCONFIG_DEFAULT_QR_CODE_PROCESS_ID = "de.metas.global_qrcodes.service.GlobalQRCodeService.qrCodeProcessId";
+
+	private final ISysConfigBL sysConfigBL = Services.get(ISysConfigBL.class);
+
+	private final AdProcessId default_qrCodeProcessId = AdProcessId.ofRepoId(sysConfigBL.getIntValue(SYSCONFIG_DEFAULT_QR_CODE_PROCESS_ID,584977));
```

Comment:
pls don't change this service. the default qr code process id is good as it is.
In almost all cases we are using.
There is only one exception, when we print auth badges.
So, for that case: "change de.metas.security.process.AuthTokenPrintQrCode process and call globalQRCodeService.createPDF with your new AD_Process_ID as param"

Reply:
done